src/node/test/sss_random.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this still required?

Reply:
No, gone now. Just wasn't done with my changes yet (more docs coming).